src/Tribe/Views/V2/Views/Traits/HTML_Cache.php

```diff
@@ -295,10 +292,14 @@ protected function should_enable_html_cache( $context ) {
 	 * Get the list of fields/input we will do replacement for HTML Cache.
 	 *
 	 * @since 5.0.0
+	 * @since TBD Deprecating for new nonce structure.
+	 *
+	 * @deprecated
```

Comment:
I think the standard is to add TBD with @deprecated

Reply:
Yes - so we can add the version on release